### ISSUE TITLE
link ldap-setup

### ISSUE THREAD

**Issue by vv01f** [fsfw-dresden/ldap-dingens#11]
wurde das vorgeschaltete setup dokumentiert?
ggf. link in die readme wäre hilfreich

**Comment by horazont** [comment]
On 03.06.2015 22:16, Wolf wrote:

> wurde das vorgeschaltete setup dokumentiert?
> ggf. link in die readme wäre hilfreich

Ich verstehe die Frage nicht, was ist mit "vorgeschaltetem Setup" gemeint?

**Comment by vv01f** [comment]
weil zuletzt aufkam, dass wir dokumentieren wollen, was (Anwendungen, Konfiguration...) alles wie wo installiert wurde. Ich nehme an man kann aus Ansible.
Einfach Link zum Abholpunkt was alles erfüllt sein muss, damit das hier läuft.

**Comment by horazont** [comment]
Ja, das Setup ist vollständig im Ansible automatisiert. Das Ansible liegt aber noch nicht in einem öffentlichen Repository.
